### ISSUE TITLE
ControlNet's loss curve is confusing.

### ISSUE THREAD

**Issue by WeiChunyu-star** [huggingface/diffusers#3591]
### Describe the bug

 I trained the ControlNet as instructed by the README documentation, but got a confusing loss curve trend. I want to make sure that this curve trend is normal.

### Reproduction

**The code I use for training is as below:**

export MODEL_DIR="runwayml/stable-diffusion-v1-5"
export OUTPUT_DIR="./outdir"

accelerate launch /home/diffusers/examples/controlnet/train_controlnet.py \
 --pretrained_model_name_or_path=$MODEL_DIR \
 --output_dir=$OUTPUT_DIR \
 --dataset_name=fusing/fill50k \
 --resolution=512 \
 --learning_rate=1e-5 \
 --validation_image "./conditioning_image_1.png" "./conditioning_image_2.png" \
 --validation_prompt "red circle with blue background" "cyan circle with brown floral background" \
 --train_batch_size=1 \
 --gradient_accumulation_steps=4

result:
![loss](https://github.com/huggingface/diffusers/assets/54472778/13c28068-b6a1-4603-b317-2b29e92a4abd)

### System Info

PyTorch 1.11
Python 3.7.16

Tesla V100 Memory=32510MiB
NVIDIA-SMI 460.67       Driver Version: 460.67       CUDA Version: 11.2

run diffusers-cli env
- `diffusers` version: 0.17.0.dev0
- Platform: Linux-5.4.0-60-generic-x86_64-with-debian-buster-sid
- Python version: 3.7.16
- PyTorch version (GPU?): 1.11.0+cu102 (True)
- Huggingface_hub version: 0.14.1
- Transformers version: 4.29.2
- Accelerate version: 0.19.0
- xFormers version: not installed
- Using GPU in script?: <examples/controlnet/train_controlnet.py>
- Using distributed or parallel set-up in script?: <no>

**Comment by WeiChunyu-star** [comment]
I restarted a training session, but the output result still didn't look nice.

![again_result](https://github.com/huggingface/diffusers/assets/54472778/e460d06f-30e8-406d-ac22-bc39142e6421)

**Comment by sayakpaul** [comment]
The loss curve is known. I would suggest tweaking the hyperparameters to see if you get expected results.

Cc: @williamberman @yiyixuxu to check if they have any insights to share.

Since this issue is not related to codebase itself, I am closing it. Feel free open such discussions here:
https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63

**Comment by williamberman** [comment]
Yes, unfortunately we do not have clean loss curves for the default controlnet training config :)
